packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -44,6 +44,24 @@ class DebuggerScreen extends Screen {
   @override
   bool showConsole(bool embed) => true;
 
+  @override
+  ShortcutsConfiguration keyboardShortcuts(BuildContext context) {
```

Comment:
nit: maybe call this buildKeyboardShortcuts?

Reply:
Done